torch/csrc/jit/tensorexpr/ir_simplifier.h

```diff
@@ -404,9 +404,9 @@ class MinTerm : public ExprNode<MinTerm> {
 };
 
 // Stmt simplification should occur in both modes.
-class TORCH_API IRSimplifierBase : public IRMutator {
+class TORCH_API PolynomialBase : public IRMutator {
```

Comment:
What is the need for this change?

Reply:
The name indicates that it is the base class of all simplifications, but the context-sensitive simplification (this PR stack) is not based on it. It currently serves as the base class of `PolynomialTransformer` and `TermExpander` so I guess `PolynomialBase` is a more appropriate name?